packages/rpc-subscriptions/src/rpc-subscriptions-json.ts

```diff
@@ -0,0 +1,26 @@
+import { RpcSubscriptionsChannel } from '@solana/rpc-subscriptions-spec';
+
+export function getRpcSubscriptionsChannelWithJSONSerialization(
+    channel: RpcSubscriptionsChannel<string, string>,
+): RpcSubscriptionsChannel<unknown, unknown> {
+    return {
+        ...channel,
+        on(type, listener, options) {
```

Comment:
Would it make sense to `Object.freeze` this?

I know technically the `RpcSubscriptionsChannel` type only has two functions (`send` and `on`) but since we're destructuring the inner object here, it may be useful to freeze it.